external_parser/example_joiner.cc

```diff
@@ -355,7 +355,8 @@ int example_joiner::process_joined(v_array<example *> &examples) {
     enqueued_time->tm_sec = joined_event->timestamp()->second();
 
     time_t enqueued_time_utc = mktime(enqueued_time);
-    std::cout << "enqueued time utc: " << enqueued_time_utc << std::endl;
+
```

Comment:
nit: we can start removing the cout statements now that we have unit tests to check stuff